shell/platform/android/io/flutter/plugin/platform/PlatformViewsController.java

```diff
@@ -623,30 +627,25 @@ private void initializePlatformViewIfNeeded(int viewId) {
 
     PlatformView platformView = factory.create(context, viewId, createParams);
     View view = platformView.getView();
-
-    // Reverse scale the view based on the screen density.
-    //
-    // Flow pixels are based on the physical resolution. For example, 1000 pixels in flow equals
-    // 500 points in Android if the display density is 2.0.
-    float density = getDisplayDensity();
-    view.setPivotX(0);
-    view.setPivotY(0);
-    view.setScaleX(1 / density);
-    view.setScaleY(1 / density);
-
     platformViews.put(viewId, view);
```

Comment:
I think we can remove `platformViews` and keep `mutatorViews`.  We don't really need a reference to the platform view itself.

Reply:
I added a change that also requires to resize the platform view when onDisplay